src/express/commands/start.js

```diff
@@ -46,8 +46,8 @@ async function installRequiredSoftwareOnRemoteMachines(ips, devnetType) {
     
     let arr = []
 
-    let deps = nodeIps.map(async(ip) => {
-        arr = ip.split("@")
+    let depTasks = nodeIps.map(async(ip) => {
```

Comment:
can we call this variable `requirementTasks` ?

Reply:
Fixed: [b658a55](https://github.com/maticnetwork/matic-cli/pull/86/commits/b658a55711f29b38b146399cdecd63cb4ecab326)